drivers/flash/flash_sam0.c

```diff
@@ -112,6 +112,23 @@ static int flash_sam0_check_status(off_t offset)
 	return 0;
 }
 
+static inline u32_t flash_sam0_read_unaligned_u32(const void *data)
```

Comment:
Instead of this, can't you just use `UNALIGNED_GET()` macro?

Reply:
I was not sure if it does the same, but if yes - that better idea.